app/helpers/settings_schedule_helper.rb

```diff
@@ -0,0 +1,115 @@
+module SettingsScheduleHelper
+  private
+
+  def settings_schedule_summary(selected_schedule, exp_table, object_class, object_name)
+    summary = [settings_schedule_basic_info(selected_schedule)]
+    summary.push(settings_schedule_filter_details(selected_schedule, exp_table))
+    summary.push(settings_schedule_object_details(selected_schedule)) if selected_schedule.sched_action[:method] == 'automation_request'
```

Comment:
the condition `if selected_schedule.sched_action[:method] == 'automation_request'` is used in multiple places, maybe we can store in common variable

Reply:
Fixed